src/pilot.c

```diff
@@ -1701,7 +1701,54 @@ void pilot_explode( double x, double y, double radius, const Damage *dmg, const
 void pilot_render( Pilot* p, const double dt )
 {
    (void) dt;
-   double scalew, scaleh;
+   double scalew, scaleh, a;
+   int i;
+   double x1, y1, x2, y2, dx, dy;
+   glColour c1, c2;
+   Vector2d* point;
+   int now;
+
+   /* Tracks. */
+   //c.a = 0.25;
+   if (array_size(p->times) > 1){
+      now = (int) ntime_get();
+
+      /* Compute the engine offset. TODO: function */
+      a  = p->solid->dir;
+      dx = p->ship->x_engine * cos(a) - p->ship->y_engine * sin(a);
+      dy = p->ship->x_engine * sin(a) + p->ship->y_engine * cos(a);
+
+      /* Lastly created control point is replaced by ship's position */
+      gl_gameToScreenCoords( &x1, &y1, p->solid->pos.x + dx, p->solid->pos.y + dy*M_SQRT1_2 + p->ship->h_engine*M_SQRT1_2 );
+      point = &p->track[ array_size(p->track)-2 ];
+      gl_gameToScreenCoords( &x2, &y2, point->x, point->y );
+      //gl_drawLine( x1, y1, x2, y2, &c );
+      gl_renderCross( x1, y1, 12., &cWhite );
+
+      /* Set the colour. TODO: function */
+      if (pilot_isFlag(p, PILOT_AFTERBURNER))
+         c1 = cGold;
```

Comment:
probably red will be better (looks less like portal), but I guess it needs testing